stl/inc/xutility

```diff
@@ -5705,12 +5705,12 @@ template <class _Diff, class _Urng>
 class _Rng_from_urng { // wrap a URNG as an RNG
 public:
     using _Ty0 = make_unsigned_t<_Diff>;
-    using _Ty1 = typename _Urng::result_type;
+    using _Ty1 = _Invoke_result_t<_Urng>;

```

Comment:
I believe that this should be `_Invoke_result_t<_Urng&>`.

WG21-N4910 [rand.req.urng] requires `invocable<G&> && unsigned_integral<invoke_result_t<G&>>`, allowing a URBG to have `R operator()() & { ... }` forbidding rvalues.